### DISCUSSION TITLE
API docs?

### DISCUSSION THREAD

**Discussion by albisserAdrian** [run-llama/LlamaIndexTS#755] [Q&A] [answered]
Anyone know where to find the API docs? From a couple of days now that the API docs are gone from the documentation site and every link ends at a "Page Not Found".

**Comment by himself65** [comment]
vercel cache used the old build output and didn't update for a while so vercel didn't catch that until recently

**Reply by himself65** [reply, marked as answer]
https://github.com/run-llama/LlamaIndexTS/pull/756

**Comment by albisserAdrian** [comment]
Thanks, I can see the API docs again.
